README.md

```diff
@@ -15,6 +15,8 @@ cmsrel  CMSSW_10_6_14
 cd  CMSSW_10_6_14/src
 cmsenv
 git cms-addpkg PhysicsTools/NanoAOD
+git cms-merge-topic andrzejnovak:DDXV2_106
```

Comment:
Can you include `git cms-addpkg  RecoBTag/Combined` here?
that is needed in the recipe

Reply:
ah nevermind, there is also a small bug in the submit_all script taht I need to change. I'll merge this PR, and fix this and the submit_all in a diff PR.